circt.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
The name `circt.sh` is not very descriptive. Maybe `build-circt.sh` as name?

Reply:
Renamed